dist/global/global.css

```diff
@@ -19,5 +19,10 @@ a:visited {
   color: var(--link-foreground-color-visited, var(--color-foreground-visited));
 }
 a:hover {
-  color: var(--link-foreground-color-visited, var(--color-foreground-visited));
+  opacity: 0.7;
```

Comment:
Wow! This looks like an evil sweeping change. Is it a mistake or is there a reason?

Reply:
This shouldn't be using opacity. There is a formula which is used to derive the final color value. Please check in figma. Actually I think we already have it defined in our tokens for this foreground colour.

Reply:
Wow, definitely a mistake. Must have been pulled in when I did the merge to `15.3.0`